testexec/README.md

```diff
@@ -0,0 +1,111 @@
+the testexec extension to testmark
+==================================
+
+See the README in the parent directory for a broader introduction to testmark as a format.
+
+"testexec" is a convention used within the testmark format to describe
+tests for executing processes.
+
+Use the [well-known hunk names](well-known-hunk-names) to write your fixtures,
+then connect it to `go test` with the connecting the code you can see in the [example code](#examples).
+It supports [autopatching and fixture generation](#autopatching),
+and the [exec hooks are configurable](#configurable-exec-hooks).
+
+
+Well-known hunk names
+---------------------
+
+The "testexec" extension is as simple as a series of well-known hunk names:
+
+Firstly, either of these two forms can be used to specify the commands to test:
+
+- "`script`" -- will feed the entire hunk of text as a script to a shell (by default, bash).
+- "`sequence`" -- will run each line as a command as its own executable (parsing args by simply whitespace split).
+
+("sequence" mode is meant to be a little less complicated to implement, and more portable, but is also a bit less flexible.
+"script" mode can do whatever the shell does -- so, handle quoting, redirections, etc -- the only challenge is then you have to know what your shell does!)
+i
```

Comment:
Remove the `i` so that the paragraph break works.

Reply:
wow, that's a nice typo.  I must've thought I was in a model editor at some point.  :face_with_spiral_eyes: